applications/crossbar/src/modules/cb_apps_store.erl

```diff
@@ -346,6 +377,40 @@ load_app(Context, AppId) ->
              )
     end.
 
+%% @private
+-spec load_app_from_master_account(cb_context:context(), ne_binary()) -> cb_context:context().
+load_app_from_master_account(Context, AppId) ->
+    {'ok', MasterAccountDb} = whapps_util:get_master_account_db(),
+    {'ok', MasterAccountId} = whapps_util:get_master_account_id(),
+    try
```

Comment:
I don't get the point of using `try` here.

Why not just get the JObjs from couch_mgr and case ListComp of [JObj] -> good; _ -> bad end ?